src/app/models/AxiosResponse.ts

```diff
@@ -0,0 +1,5 @@
+export interface AxiosResponse {
+  data: any,
```

Comment:
Replace any type with the use of Generic Type
https://www.typescriptlang.org/docs/handbook/2/generics.html

Example:
`export interface AxiosResponse<RecordType> {
  data: RecordType[],
  hasErrors: boolean,
  errorMessage: string,
}
`